lib/std/jsfetch.nim

```diff
@@ -70,7 +70,7 @@ func newRequest*(url: cstring): Request {.importjs: "(new Request(#))".}
 func clone*(self: Response or Request): Response {.importjs: "#.$1()".}
   ## https://developer.mozilla.org/en-US/docs/Web/API/Response/clone
 
-proc text*(self: Body or Request): Future[cstring] {.importjs: "#.$1()".}
+proc text*(self: Body or Response): Future[cstring] {.importjs: "#.$1()".}
```

Comment:
`proc text*(self: Body or Response):` vs `proc text*(self: Response):` : which one is correct? can you point to some docs or code usage justifying `proc text*(self: Body or Response):` ?

Reply:
Official documentation is linked. [`Body is implemented by both Request and Response.`](https://developer.mozilla.org/en-US/docs/Web/API/Body)

Reply:
The 3 should use the proc ?.

Reply:
* one option is `proc text*(self: Response or Request):`
* another option is: `type Body = Response or Request` and then using:
```nim
proc text*(self: Body):
```

which one is better deserves more thought

Reply:
> * another option is: `type Body = Response or Request`

No, because the `Response` contains a type `Body`, 
but a `Response` does not contain a `Response`, 
kinda like `Body` is a type by itself too.